include/ultra64/rcp.h

```diff
@@ -39,6 +39,24 @@
 #define VI_X_SCALE_REG          0x04400030  //VI x-scale
 #define VI_Y_SCALE_REG          0x04400034  //VI y-scale
 
+// VI_CONTROL_REG (read) bits
+#define VI_CTRL_TYPE_16             0x00002 // [1:0] pixel size: 16 bit
+#define VI_CTRL_TYPE_32             0x00003 // [1:0] pixel size: 32 bit
+#define VI_CTRL_GAMMA_DITHER_ON     0x00004 // 2: default = on
+#define VI_CTRL_GAMMA_ON            0x00008 // 3: default = on
+#define VI_CTRL_DIVOT_ON            0x00010 // 4: default = on
+#define VI_CTRL_SERRATE_ON          0x00040 // 6: on if interlaced
+#define VI_CTRL_ANTIALIAS_MASK      0x00300 // [9:8] anti-alias mode
+#define VI_CTRL_ANTIALIAS_MODE_0    0x00000 // Bit [9:8] anti-alias mode: AA enabled, resampling enabled, always fetch extra lines
+#define VI_CTRL_ANTIALIAS_MODE_1    0x00100 // Bit [9:8] anti-alias mode: AA enabled, resampling enabled, fetch extra lines as-needed
+#define VI_CTRL_ANTIALIAS_MODE_2    0x00200 // Bit [9:8] anti-alias mode: AA disabled, resampling enabled, operate as if everything is covered
+#define VI_CTRL_ANTIALIAS_MODE_3    0x00300 // Bit [9:8] anti-alias mode: AA disabled, resampling disabled, replicate pixels
+#define VI_CTRL_PIXEL_ADV_MASK      0x01000 // [15:12] pixel advance mode
+#define VI_CTRL_PIXEL_ADV_1         0x01000 // Bit [15:12] pixel advance mode: ?
+#define VI_CTRL_PIXEL_ADV_2         0x02000 // Bit [15:12] pixel advance mode: ?
```

Comment:
n64brew only mentions 0b11 and the code only mentions 0b11 so probably these two can go?

Reply:
These other modes do exist to some capacity, afaik, so I think it's better to maintain a complete set of defines.
Also notably, the iQue player needs different pixel advance settings or the output is corrupted

Reply:
why only the two low bits then?

Reply:
Because I copied them from ultralib and never noticed it wasn't complete 😎 
I've changed it to one macro with an argument to generate all values.